README.rst

```diff
@@ -37,7 +37,7 @@ It has many features, but here are some of the nicer ones:
 * Syntax highlighting for almost any programming language or markup
 * Multilingual sites, `translated to 50 languages.`__
 * Doesn't reinvent wheels, leverages existing tools.
-* Python 3.4+ compatible.
```

Comment:
Fixed (to say 3.5) in commit https://github.com/getnikola/nikola/commit/8cc31e0230a7209cc0bca558a9d085b2e179fe8c.